src/icalendar/prop/__init__.py

```diff
@@ -911,6 +911,361 @@ def from_jcal(cls, jcal_property: list) -> Self:
         )
 
 
+class vAdr:
+    """vCard ADR (Address) structured property per :rfc:`6350` Section 6.3.1.
+
+    The ADR property represents a delivery address with 7 semicolon-separated fields:
```

Comment:
Numerals between zero and nine, inclusive, in narrative text should be spelled out. See https://learn.microsoft.com/en-us/style-guide/numbers.

```suggestion
    The ADR property represents a delivery address with seven semicolon-separated fields:
```